src/main/resources/application.yaml

```diff
@@ -35,3 +35,8 @@ jwt:
     auth: Authorization
     expire-time-range: 3600000
 
+api:
+  mode: dev
```

Comment:
Try to parse url to determine if it's dev or production, and avoid this setting

Reply:
fix